src/main/static/stylesheets/features.scss

```diff
@@ -0,0 +1,234 @@
+.tenancies-list {
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  gap: 40px;
+  justify-content: center;
+}
+
+.features-connect {
+  display: flex;
+  justify-content: center;
+  margin: 2rem 0 6.5rem;
+
+  &__button {
+    width: 390px;
+  }
+}
+
+.features-faqs {
+  background-color: var(--background-light-2);
+  font-weight: 700;
+  padding: 40px 140px;
+  width: 100%;
+}
+
+.features-banner {
+  background: var(--background-gradient-banner);
+  color: var(--color-text-dark-1);
+  padding: 55px 2rem;
+
+  &__list {
+    align-items: center;
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    gap: 70px;
+    justify-content: center;
+
+    &__item-details {
+      max-width: 260px;
+
+      &__title,
+      .image-icon,
+      &__paragraph {
+        margin-bottom: 18px;
+      }
+
+      &__title {
+        font-weight: 700;
+      }
+
+      &__main-icon {
+        height: 30px;
+        width: 30px;
+
+        g path {
+          fill: var(--color-icon-dark-2);
+        }
+
+        path {
+          stroke: var(--color-icon-dark-2);
+        }
+      }
+
+      &__learn-button {
+        &__text {
+          margin-right: 20px;
+        }
+      }
+    }
+  }
+}
+
+.tenancy-info {
+  background-color: var(--color-highlight-4);
+  border: 1.5px solid var(--color-highlight-3);
+  border-radius: 22px;
+  max-width: 350px;
+  min-height: 430px;
+  padding: 36px 40px;
+
+  hr {
+    background-color: var(--color-highlight-5);
+    height: 1px;
+    margin: 10px 0;
+  }
+
+  &__level,
+  &__service {
+    color: var(--color-highlight-2);
+    font-size: var(--font-size-body-5);
+    font-weight: 700;
+  }
+
+  &__level {
+    font-size: var(--font-size-body-5);
+    text-transform: uppercase;
+  }
+
+  &__tenancy {
+    font-weight: 700;
+    text-transform: uppercase;
+  }
+
+  &__features {
+    align-items: center;
+    display: flex;
+    min-height: 165px;
+
+    &__content {
+      &__header {
+        color: var(--color-text-light-7);
+        display: flex;
+        font-weight: 700;
+        gap: 8px;
+        margin-bottom: 12px;
+
+        &__details {
+          font-size: var(--font-size-body-5);
+        }
+      }
+
+      &__list {
+        &__item {
```

Comment:
try not to nest items this deeply (this is for .tenancy-info__features__content__list__item)

you don't need every child to be nested, you can have .tenancy-info__features, .tenancy-info__content, .tenancy-info__list, .tenancy-info__items.

I'm trying to stick to 1 nesting in most cases, occasionally 2 if it really warrants it. 

if something within a component is more complex and needs nesting, consider just giving it a different parent name

e.g. .component__sub-component__child -> .sub-component__child